API.md

```diff
@@ -215,3 +215,35 @@ instead return a [Joi ValidationError object](https://joi.dev/api/?v=17.9.1#vali
   updatedDt: string (date),
 }
 ```
+
+## Accounts
+
+### PUT `/tags`
+
+- Headers: Authorization: Bearer \<JWT from FusionAuth>
+- Request Body
+
+```
+{
+  addTags: [string]
+  removeTags: [string]
+}
+```
+
+- Response
+
+```
+{} (empty response with 200 code indicates success)
+```
+
+### GET `/signups`
```

Comment:
Looks like this is singular?

Reply:
I also think both of these would be clearer if they were written as `account/tags` and `account/signup`, as we do with `/directive/archive/:archiveId` above, even though it's in the "Directives" section of the docs.